File: task-03/Java/Sub_Task2.java
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.Paths;

public class Sub_Task2 {
    public static void main(String[] args) {
        // Define file paths
        Path inputFile = Paths.get("input.txt");
        Path outputFile = Paths.get("output.txt");

        try {
            // Read the content from input.txt
            String content = Files.readString(inputFile);

            // Write the content to output.txt
            Files.writeString(outputFile, content);

            System.out.println("File content copied successfully from input.txt to output.txt.");
        } catch (IOException e) {
            System.err.println("An error occurred: " + e.getMessage());
        }
    }
}

File: task-03/Ruby/Sub-Task2.rb
# Define file paths
input_file = 'input.txt'
output_file = 'output.txt'

begin
  # Read the content from input.txt
  content = File.read(input_file)

  # Write the content to output.txt
  File.write(output_file, content)

  puts "Content successfully copied from #{input_file} to #{output_file}."
rescue Errno::ENOENT => e
  puts "An error occurred: #{e.message}"
rescue IOError => e
  puts "An I/O error occurred: #{e.message}"
end
